pybabel missing from requirements files?

I created a fresh virtualenv, ran `make build` and was told that "`.state/docker-build` is up to date.", then tried to run `make build-mos` (following up #6535) and got this error:

~~~
$ make build-mos
[redacted path]/.state/env/bin/pybabel extract \
	-F babel.cfg \
	--copyright-holder="PyPA" \
	--msgid-bugs-address="https://github.com/pypa/warehouse/issues/new" \
	--project="Warehouse" \
	--output="warehouse/locale/messages.pot" \
	warehouse
/bin/sh: 1: [redacted path]/.state/env/bin/pybabel: not found
make: *** [Makefile:178: compile-pot] Error 127
~~~

I see `pybabel` isn't in any of our requirements files -- am I missing something? @woodruffw did you add the dependency per https://warehouse.readthedocs.io/development/patterns/#adding-new-dependencies ?

--- a/Makefile
+++ b/Makefile
@@ -174,7 +174,7 @@ purge: stop clean
 stop:
 	docker-compose down -v
 
-compile-pot:
+compile-pot: .state/env/pyvenv.cfg
 	$(BINDIR)/pybabel extract \
 		-F babel.cfg \
 		--copyright-holder="PyPA" \
@@ -183,19 +183,19 @@ compile-pot:
 		--output="warehouse/locale/messages.pot" \
 		warehouse
 
-init-po:
+init-po: .state/env/pyvenv.cfg
 	$(BINDIR)/pybabel init \
 		--input-file="warehouse/locale/messages.pot" \
 		--output-dir="warehouse/locale/" \
 		--locale="$(L)"
 
-update-po:
+update-po: .state/env/pyvenv.cfg
 	$(BINDIR)/pybabel update \
 		--input-file="warehouse/locale/messages.pot" \
 		--output-file="warehouse/locale/$(L)/LC_MESSAGES/messages.po" \
 		--locale="$(L)"
 
-compile-po:
+compile-po: .state/env/pyvenv.cfg
 	$(BINDIR)/pybabel compile \
 		--input-file="warehouse/locale/$(L)/LC_MESSAGES/messages.po" \
 		--directory="warehouse/locale/" \

